wunderio/next4drupal-project
# NEX-45: nested paragraphs example

### "It's paragraphs all the way down!" 😨 

This PR adds an example of nested paragraph to the backend, frontend and migration setup.

We are adding: 

* an accordion paragraph, which has to contain:
   * accordion_item paragraphs, which can contain:
     * formatted text paragraph
     * video paragraph
     * image paragraph
     
To support this in this PR we have:

* modified zod schema to match the paragraphs
* added the Radix accordion npm package: https://www.radix-ui.com/docs/primitives/components/accordion
* tweaked styles
* added migration of the new accordion paragraph type, accordion items, and variable content in each of the accordion items to demostrate the variety of paragraphs that can be added

### Test instructions:

* run the usual big command: https://github.com/wunderio/next4drupal-project#%EF%B8%8F-quick-one-command-setup
* visit the "Paragraph example en" page, and verify that it contains the new accordion paragraph with the included second and third level paragraphs in it.
* visit the corresponding translated pages, check that the text is translated correctly

### Screenshots

Backend:
![image](https://user-images.githubusercontent.com/185412/222430631-3336b4d3-1b61-43fb-88ec-5d87ccda2a40.png)

Frontend: 
![image](https://user-images.githubusercontent.com/185412/222430372-0fc61a8c-17e3-41ee-aa10-55822951bce3.png)
